homeassistant/components/hive/__init__.py

```diff
@@ -13,14 +13,17 @@
     CONF_USERNAME,
 )
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers.discovery import async_load_platform
 from homeassistant.helpers.dispatcher import async_dispatcher_connect, dispatcher_send
 from homeassistant.helpers.entity import Entity
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_AVAILABLE = "available"
+CONF_MODE = "mode"
 DOMAIN = "hive"
 DATA_HIVE = "data_hive"
+ENTITY_LOOKUP = {}
```

Comment:
Store this dict in hass.data. It's not a constant.

Reply:
Fixed in latest commit